src/trustyai/explainers/lime.py

```diff
@@ -260,21 +257,35 @@ def __init__(self, samples=10, **kwargs):
                 process.
             * trackCounterfactuals : bool
                 (default= ``False``) Keep track of produced byproduct counterfactuals during LIME run.
+            * samples: int
+                (default=``300``) Number of samples to be generated for the local linear model training.
+            * encoding_params: Union[list, tuple]
+                (default=``(0.07, 0.3)``) Lime encoding parameters, as a tuple/list of two float numbers:
+                - encoding_params[0] is the width of the Gaussian filter for clustering number features.
+                - encoding_params[1] is the threshold for clustering number features.
 
         """
         self._jrandom = Random()
         self._jrandom.setSeed(kwargs.get("seed", 0))
-
+        ep = kwargs.get("encoding_params", (0.07, 0.3))
         self._lime_config = (
             LimeConfig()
             .withNormalizeWeights(kwargs.get("normalise_weights", False))
             .withPerturbationContext(
                 PerturbationContext(self._jrandom, kwargs.get("perturbations", 1))
             )
-            .withSamples(samples)
-            .withEncodingParams(EncodingParams(0.07, 0.3))
-            .withAdaptiveVariance(True)
+            .withSamples(kwargs.get("samples", 300))
```

Comment:
Minor note, currently the default value is `10` I don't know if the impact of the PR in terms of execution time/doc.
Btw I think it is safe

Reply:
yeah, the current value of _10_ is suboptimal anyway (the default in the java impl is _300_).